app/models/user.php

```diff
@@ -108,27 +106,81 @@ public function login()
         }
 
         $_SESSION['user_id'] = $user['id'];
-        $_SESSION['username'] =$user['username'];
+        $_SESSION['first_name'] =$user['first_name'];       
     }
 
-    public function is_password_match()
+    public function isPasswordMatch()
     {
         return $this->password == $this->confirm_password;
     }
 
-    public function is_username_exist()
+    public function isUsernameExist()
     {
         $db = DB::conn();
-        $username_exist = $db->row("SELECT username FROM user WHERE username = ?", array($this->username));
+        $username_exist = $db->row("SELECT username FROM user WHERE 
+                                    username = ?", array($this->username));
 
         return !$username_exist;
     }
 
-       public function is_email_exist()
+    public function isEmailExist()
     {
         $db = DB::conn();
-        $username_exist = $db->row("SELECT email FROM user WHERE email = ?", array($this->email));
+        $params = array($this->email, 
+                        $_SESSION['user_id']
+                    );
 
-        return !$username_exist;
+        $email_exist = $db->row("SELECT email FROM user WHERE
+                                 email = ? && id != ?", $params);
+
+        return !$email_exist;
+    }
+
+    public static function getUsername($user_id)
+    {
+        $db = DB::conn();
+        $user = $db->row("SELECT username FROM user WHERE
+                            id = ?", array($user_id));    
+
+        return $user['username'];
+    }
+
+    public static function get()
+    {
+        $db = DB::conn();
+        $row = $db->row("SELECT * FROM user WHERE
+                        id = ?", array($_SESSION['user_id']));    
+        
+        if (!$row) {
+            throw new RecordNotFoundException('no record found');
+        }
+
+        return new self($row);
+    }
+
+    public function update()
+    {
+         if (!$this->validate()) {
+            throw new ValidationException('Invalid user credentials');
+        }
+
+        try {
+            $db = DB::conn();
+
+            $db->begin();
+            $params = array(
+                        'first_name' => $this->first_name,
+                        'last_name' => $this->last_name,
+                        'email' => strtolower($this->email)
+                        );
+            $db->update('user', $params,
+                        array('id'=>$_SESSION['user_id'])   
+                    );
+            $db->commit();
+
+        }catch(Exception $e) {
```

Comment:
Please add space after the curly brace and before the catch.
Apply the same principle as this comment: https://github.com/adrianneperedo/board/pull/2/files#r26102656